14/umbraco-cms/fundamentals/backoffice/sections.md

```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
 * Extension Insights
 * Webhooks
 
-The **Settings** section of the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
+The **Settings** section in the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
 
 ## Packages
 
 In this section, you can browse and install packages into your Umbraco solution. You can also get an overview of all the installed packages as well as uninstall packages you no longer need.
```

Comment:
You can't install packages directly from this section, so I think we should take that out of this section!

Reply:
Updated the text.